tasks/rh-subscription.yml

```diff
@@ -55,10 +55,11 @@
   rhsm_repository:
     name: "{{ item }}"
     state: "enabled"
+  ignore_errors: True
```

Comment:
Instead of ignoring errors I'd remove tools channel. Patch for RHV docs removing tools channel should be merged today

Reply:
Okay, will update it merge it and do the build so we can have it for the demo.